includes/Utilities/Tracker.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+ *
+ * This source code is licensed under the license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @package FacebookCommerce
+ */
+
+namespace SkyVerge\WooCommerce\Facebook\Utilities;
+
+defined( 'ABSPATH' ) or exit;
+
+/**
+ * Class for adding diagnostic info to WooCommerce Tracker snapshot.
+ *
+ * See https://woocommerce.com/usage-tracking/ for more information.
+ *
+ * @since %VERSION%
```

Comment:
I'm using this `%VERSION%` as a temporary placeholder, will need to be a real version when this ships. 

If there's automation for updating these let me know, otherwise I'll update when we know what version we're targeting. Likely v2.3.4.